### ISSUE TITLE
trayscale on steamdeck error

### ISSUE THREAD

**Issue by st0rm3rd** [DeedleFake/trayscale#98]
Getting following error after installing Trayscale on the SD and following the instructions:
(127)(deck@steamdeck ~)$ sudo tailscale set --operator=$USER
sudo: tailscale: command not found
Doesn't seem to have been installed despite launching from Dolphine?

**Comment by DeedleFake** [comment]
That's correct. Unfortunately, Tailscale requires stuff too low level to allow it to be installable as a Flatpak. Ironic, considering that I had originally started working on this so that I could use it on my Steam Deck... As far as I am aware, there's nothing I can do about it. Sorry. Installing Tailscale separately, such as via `pacman`, should work, but I believe that it will get deleted the next time that you update SteamOS and will then need to be reinstalled and reconfigured.

**Comment by St0rm3rt** [comment]
Thanks Deedlefake. That's too bad. Would have been nice.
When I read the comments in Discover, I understood it to be working on the SD, but I guess me it was not referring to SD.
